client/src/document/index.scss

```diff
@@ -4,3 +4,7 @@
   top: 0;
   right: 5px;
 }
+
+#content h2 a {
+  color: rgb(51, 51, 51);
```

Comment:
@schalkneethling This isn't the best place to do something like this. But I don't think it should go into mdn-minimalist either. I'm not to worried at the moment. If we accidentally forget this when we go to fiori ui, we'll just put it back.